pulsar-testclient/src/main/java/org/apache/pulsar/proxy/socket/client/PerformanceClient.java

```diff
@@ -166,7 +168,9 @@ public void runPerformanceTest(long messages, long limit, int numOfTopic, int si
             String topicName, String authPluginClassName, String authParams) throws InterruptedException, FileNotFoundException {
         ExecutorService executor = Executors.newCachedThreadPool(new DefaultThreadFactory("pulsar-perf-producer-exec"));
         HashMap<String, Tuple> producersMap = new HashMap<>();
-        String produceBaseEndPoint = baseUrl + "ws/producer" + topicName;
+        String replacedTopicName = topicName.replace("://", "/");
```

Comment:
I think there has a method `getRestPath` in the TopicName do the same thing with you.

Reply:
Thank you for the advice. 
I fixed it.